Source/Anchorage.swift

```diff
@@ -537,7 +553,7 @@ public struct EdgeGroup {
     public var trailing: NSLayoutConstraint
 
     public var horizontal: [NSLayoutConstraint] {
-        return [leading, trailing].flatMap { $0 }
+        return [leading, trailing]
```

Comment:
Oh, I actually just rewrote and removed this change before I read this, as I added it by accident and it doesn't have anything to do with priority. I can add it back though if you like it (and all the other places it does this)